src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidator.java

```diff
@@ -99,9 +106,19 @@ private void validateUpdateDetails(OverseasEntitySubmissionDto overseasEntitySub
     }
 
     private void validateFullRemoveDetails(OverseasEntitySubmissionDto overseasEntitySubmissionDto, Errors errors, String loggingContext) {
-        removeValidator.validateFull(overseasEntitySubmissionDto, errors, loggingContext);
 
-        validateUpdateDetails(overseasEntitySubmissionDto, errors, loggingContext);
+        var updateDto = overseasEntitySubmissionDto.getUpdate();
+
+        if (updateDto == null) {
+            String errorMessage = ValidationMessages.NOT_NULL_ERROR_MESSAGE.replace("%s", OverseasEntitySubmissionDto.UPDATE_FIELD);
+            setErrorMsgToLocation(errors, OverseasEntitySubmissionDto.UPDATE_FIELD, errorMessage);
+            ApiLogger.infoContext(loggingContext, errorMessage);
+        } else {
+            validateUpdateDetails(overseasEntitySubmissionDto, errors, loggingContext);
```

Comment:
the validateUpdateDetails used to run after removeValidator.validateFull and didn't have a updateDto == null check around it, I would move it back underneath the removeValidator.validateFull without the updateDto == null check

Reply:
We require this method call to be in a condition as the first line of the method calls a method on the update dto which will throw an npe.

Reply:
I missed the possible npe which could happen on kine 1 of the validateUpdateDetails method so i'm happy with this.